src/main/resources/avro/assaymetadata.avdl

```diff
@@ -0,0 +1,146 @@
+@namespace("org.ga4gh.models")
+
+/**
+This protocol defines metadata objects describing experimental or analytical procedures, performed on biological objects (e.g. biosamples) or readouts from experimental procedures.
+*/
+
+protocol Assaymetadata {
+
+  import idl "common.avdl";
+  import idl "metadata.avdl";
+
+  /**
+  An analysis contains an interpretation of one or several experiments.
+  (e.g. SNVs, copy number variations, methylation status) together with
+  information about the methodology used.
+  */
+  record Analysis {
+
+    /**
+    Formats of id | name | description | accessions are described in the
+    documentation on general attributes and formats.
+    */
+    string id;
+
+    union { null, string } name = null;
+
+    union { null, string } description = null;
+
+    /**
+    The time at which this record was created.
+    Format: :ref:`ISO 8601 <metadata_date_time>`
+    */
+    union { null, string } createDateTime = null;
```

Comment:
Michael Baudis <notifications@github.com> writes:
> This went through n iterations. The most verbose was recordCreateTime I think,
> explicitly scoping the meaning to the record, not the described object (i.e.
> record of a biosample, not sampling time).

recordCreateTime is clearer than the current `created' or
`createDateTime'.  In the other objects, it appears that it's
intention is like unix st_ctime, but that is only understood by
oral tradition.

However this is just a plea to make it consistent an then go
back an fix it schema wide.   

> This should be settled ASAP, schema wide.

Nothing happens ASAP in GA4GH.